src/features/common/Layout/Footer/Footer.module.scss

```diff
@@ -12,6 +12,7 @@
   flex-wrap: wrap;
   margin-bottom: 80px;
   justify-content: center;
+  margin-top: 20px;
```

Comment:
Remove the 20px on the container

and replace button instead of div

Reply:
This is so that each logo gets the margin 

<img width="320" alt="Screenshot 2021-03-25 at 5 41 32 PM" src="https://user-images.githubusercontent.com/25605193/112471274-b1576080-8d91-11eb-913f-9842b30f9a5a.png">